### DISCUSSION TITLE
How can I parse this xml?

### DISCUSSION THREAD

**Discussion by Martin-1182** [orchestral/parser#109] [Q&A]
### Hi, how can I parse this xml?

```
<DELIVERY>
  <DELIVERY_ID>GLS</DELIVERY_ID>
  <DELIVERY_PRICE>1.99</DELIVERY_PRICE>
  <DELIVERY_PRICE_COD>2.84</DELIVERY_PRICE_COD>
</DELIVERY>
<DELIVERY>
  <DELIVERY_ID>DPD</DELIVERY_ID>
  <DELIVERY_PRICE>3.30</DELIVERY_PRICE>
  <DELIVERY_PRICE_COD>4.14</DELIVERY_PRICE_COD>
</DELIVERY>
```

**I'll try it this way, but I won't get DELIVERY_PRICE_COD into the array**

`DELIVERY(DELIVERY_ID=DELIVERY_PRICE)>DELIVERIES`

**Comment by mnsrbk** [comment]
$data = $xml->parse([
      'deliveries' => ['uses' => 'DELIVERY[DELIVERY_ID>id,DELIVERY_PRICE>price,DELIVERY_PRICE_COD>code]'] ]);
dd($data);

**Reply by Martin-1182** [reply]
@mnsrbk @crynobone 
Hi, I got it like this. The record you sent me doesn't work. Can you help me?

'shop' => [
                    'uses' => 'SHOPITEM[ITEM_ID,PRODUCTNAME,PRODUCT,DESCRIPTION,URL,IMGURL,IMGURL_ALTERNATIVE(@=@),PRICE_VAT,VAT,MANUFACTURER,CATEGORYTEXT,EAN,DELIVERY_DATE,ITEMGROUP_ID,HEUREKA_CPC,CUSTOM_LABEL_2,DELIVERY(DELIVERY_ID=DELIVERY_PRICE)>DELIVERIES,PARAM(PARAM_NAME=VAL)>VARIATIONS]'
                ],
